reference_data/management/commands/utils/update_utils.py

```diff
@@ -55,6 +55,61 @@ def get_gene_for_record(self, record):
             raise ValueError('Gene "{}" not found in the GeneInfo table'.format(gene_id or gene_symbol))
         return gene
 
+    def update_records(self, file_path=None):
```

Comment:
this is just copy paste from below, the only difference is the `reference_data_handler` arg has been changed to `self`